src/Plugin/EntityBrowser/Display/Modal.php

```diff
@@ -177,7 +173,7 @@ public function addAjax(array &$form) {
     );
 
     $form['actions']['submit']['#ajax'] = array(
-      'callback' => array(get_class($this), 'widgetAjaxCallback'),
+      'callback' => array($this, 'widgetAjaxCallback'),
```

Comment:
This approach was causing problems when form was saved to cache ($this being serialized).